LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-  Copyright 2006-2008 Michael Hudson mwh@python.net
+  Copyright 2006-2008 Michael Hudson <mwh@python.net>
```

Comment:
While adding copyright notices, perhaps both sections of this license file should have an additional notice that covers the recent changes? Say,

> Copyright 2006-2020 The pydoctor Authors

Then add an AUTHORS file with the full list?

Or list everyone right here, like in https://github.com/twisted/twisted/blob/trunk/LICENSE

Reply:
I'd prefer to have a low-maintenance solution. Git already keeps track of author information, in much more detail, and is always up-to-date. Would it be sufficient to list the main author(s) in LICENSE and add a line like this?

> Copyright 2006-2020 Various contributors (see Git history)

Reply:
From a copyright perspective the contributors (human beings that did the work) and authors (copyright holders) may be different, though I've no real reason to think that is the case here. The specific wording I suggested was derived from the Go license, which seems a fairly common pattern:

https://github.com/golang/go/blob/a4ba411b19fa0111a3c8fe059fcf5489d3bd6bbf/LICENSE#L1

That project maintains separate [CONTRIBUTORS](https://github.com/golang/go/blob/master/AUTHORS) and [AUTHORS](https://github.com/golang/go/blob/master/AUTHORS) files.

hyper-h2 is another MIT-licensed project:

https://github.com/python-hyper/hyper-h2/blob/06aa5c9e63c4a8e0146c33f0abf14e8f7caf3945/LICENSE#L3

And Pip is similar:

https://github.com/pypa/pip/blob/e774eccf46f8465db707de108ea41a9cd9a41b66/LICENSE.txt#L1

In the absence of any clear Twisted policy on this, I think your suggestion is a fine one. What seems important to me is to avoid incorrect information, which a copyright notice that doesn't mention the additional contributors would be.